packages/pwa-buildpack/lib/WebpackTools/plugins/RootComponentsPlugin.js

```diff
@@ -159,10 +163,10 @@ class RootComponentsPlugin {
                                 );
                                 importerSources[
                                     key
-                                ] = `() => import(/* webpackChunkName: "${key}" */'./${relative(
+                                ] = `function () { import(/* webpackChunkName: "${key}" */'./${relative(
```

Comment:
This function should *return* the result of `import()`, as the arrow function did.